src/main/java/org/myrobotlab/service/abstracts/AbstractMotor.java

```diff
@@ -26,9 +26,7 @@
 
 package org.myrobotlab.service.abstracts;

```

Comment:
changes to motor control should be done in a different pull request. please.

Reply:
These file only get changed updated, because I'm keeping it in sync with develop.  These are merge conflicts that have been resolved, because changes occurred on this file and were pushed into develop from another pr.